### ISSUE TITLE
The Link bit should complement when the ALU overflows

### ISSUE THREAD

**Issue by wilsonsamm** [gordonjcp/pdp8#1]
This issue concerns cpu.c:51. What the PDP-8 does when an addition overflows, is the Link bit flips. If it was 1 then it turns to 0 and vice-versa. But I have forked your project and pushed a fix to my own copy; I think you have to merge that or something...?

**Comment by gordonjcp** [comment]
Yes, you could send me a pull request.  I'm amazed that someone found my daft little project from nearly 15 years ago interesting enough to hack on!
